client-spark/spark3/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -137,6 +137,8 @@ public class RssShuffleManager extends RssShuffleManagerBase {
   private Map<Integer, ShuffleHandleInfo> shuffleIdToShuffleHandleInfo;
   /** Whether to enable the dynamic shuffleServer function rewrite and reread functions */
   private boolean rssResubmitStage;
+
+  private boolean taskFailRetry;
```

Comment:
taskFailRetry -> taskReattemptOnFailures

Is is better?

Reply:
I don't think this is only valid for task write failure, in the future, it could be applied on the not-enough-capacity disk servers

Reply:
`taskFailureRetryEnabled`